src/SIM/SIMbase.C

```diff
@@ -1285,7 +1285,7 @@ bool SIMbase::solutionNorms (const TimeDomain& time,
           if (ssol[k].empty())
             norm->getProjection(k).clear();
           else
-            pch->extractNodeVec(ssol[k],norm->getProjection(k),nCmp);
+            this->extractPatchSolution(ssol[k],norm->getProjection(k),lp-1,nCmp,1);
```

Comment:
I'm not 100% sure about this change. Have a faint gut feeling there was something, sometime, when I had to use the extractNodeVec. But can't remember it now. Does it pass all the test? (They may not cover all aspects of norm evaluation though..)